### PR TITLE
RabbitMQ: Notes about delay queues and obsolete timeout manager

### PR DESCRIPTION
**Pull request by ramonsmits**
Several customers recently asked about how the creation of delay queues can be prevented and asked about the timeout manager with RabbitMQ.

### CODE REVIEW

File: transports/rabbitmq/delayed-delivery.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ redirects:
 
 In Versions 4.3 and above, the RabbitMQ transport no longer relies on the [timeout manager](/nservicebus/messaging/timeout-manager.md) to implement [delayed delivery](/nservicebus/messaging/delayed-delivery.md). Instead, the transport creates infrastructure inside the broker which can delay messages using native RabbitMQ features.
 
+Note: The creation of **nsb.delay-level00** to **nsb.delay-level27** cannot be avoided when using NServiceBus with RabbitMQ.
```
Comment (bording) [inline transports/rabbitmq/delayed-delivery.md]: I don't like this note, because it starts talking about concepts that haven't been introduced yet.

**Comment by danielmarbach** [comment]
I'm with @bording on this one. Let's close it

**Comment by SzymonPobiega** [comment]
@ramonsmits can you close it if you agree?

**Comment by danielmarbach** [comment]
@ramonsmits based on the agreement with @SzymonPobiega and @bording and for a long time no interaction I'm taking the liberty to close this PR. Feel free to reopen if you want to re-engage the conversation.

**Pull request closed without merging**